lib/Gym/space_handler.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Gyx.Gym.Utils do
+  @moduledoc """
+  This module contains auxiliary functions to achieve full
+  compatibility with Gym, including functions to obtain
+  Gyx space representations from Gym space specs.
+  """
+  @space_types [{~r/Discrete\((?<n>\d+)\)/, :discrete}]
+
+  @doc """
+  This function takes the __repr__ response from Gym spaces
+  and creates an equivalent Gyx Space struct
+  """
+  def gyx_space(gym_space_string) do
+    gym_space_string |> parse |> create_space
+  end
+
+  def parse(gym_space_string) do
+    {regex, type} =
+      Enum.find(@space_types, {nil, :unknown}, fn {regex, _} ->
+        Regex.match?(regex, Kernel.inspect(gym_space_string))
+      end)
+
+    {type, Regex.named_captures(regex, Kernel.inspect(gym_space_string))}
```

Comment:
creo que las funciones del modulo Kernel están disponibles en todos lados sin tener que especificar.. la otra es, porque necesitas regresar el resultado del inspect? que contiene gym_space_string?

Reply:
It comes from a Python `str` which is not automatically translated to an Elixir String.

Reply:
and it's an elixir what?  I think I remember this.. is it a list of raw numbers or something?  
I'm just concerned inspect might be adding some unnecessary overhead because of what `Kernel.inspect` does, and there should be a more specialized way of translating the characters into an elixir string.

Not sure what the correct solution is though..

Reply:
1b8af74122f653c1278617b7839bf41fddd8c60e no inpect anymore